Libplanet/Store/IStore.cs

```diff
@@ -85,6 +65,59 @@ void PutBlock<T>(Block<T> block, Address addressesMask)
             AddressStateMap states
         );
 
+        /// <summary>
+        /// Gets a <see cref="Block{T}.Hash"/> which has the state of the
+        /// <paramref name="address"/>.
+        /// </summary>
+        /// <param name="namespace">The namespace to get
+        /// <see cref="Block{T}.Hash"/>.</param>
+        /// <param name="address">The <see cref="Address"/> to find the
+        /// <see cref="Block{T}.Hash"/>.</param>
+        /// <param name="offsetIndex">The offset at which to begin looking for
+        /// a block hash.</param>
```

Comment:
This parameter and `ForkAddressStateBlockHash()` method's `branchPointIndex` takes a block index instead of a block hash, but is there any intention on this?  To me, these parameters can take a block hash and it's less error-prone as block hashes are hard to overlap by mistake (whereas block indexes are easily overlapped and the same index can refer to different blocks depending on branch).

Reply:
Oh, I got it.  These parameters cannot take a block hash because these operations depend on `<=`/`>=` comparisons.